pkg/builder/builder.go

```diff
@@ -111,9 +111,17 @@ func buildVolumeMounts(vdb *vapi.VerticaDB) []corev1.VolumeMount {
 	// container. Otherwise, you will get multiple mount points shared the same
 	// path, which will prevent any pods from starting.
 	if vdb.Spec.Local.DataPath != vdb.Spec.Local.DepotPath {
-		volMnts = append(volMnts, corev1.VolumeMount{
-			Name: vapi.LocalDataPVC, SubPath: vdb.GetPVSubPath("depot"), MountPath: vdb.Spec.Local.DepotPath,
-		})
+		if vdb.IsDepotVolumeEmptyDir() {
+			// If depotVolume is EmptyDir, the depot is stored in its own 'emptyDir' volume
+			vmts := []corev1.VolumeMount{
+				{Name: vapi.DepotMountName, MountPath: vdb.Spec.Local.DepotPath},
+			}
+			volMnts = append(volMnts, vmts...)
```

Comment:
Could we combine these statements?
```suggestion
			volMnts = append(volMnts, corev1.VolumeMount{
				Name: vapi.DepotMountName, MountPath: vdb.Spec.Local.DepotPath,
			})
```